CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+You want to contribute? We really appreciate that!
+Please have a look at this list to faciliate your contribution:
```

Comment:
typo: faciliate -> fcilitate

Reply:
You mean facilitate ;-). Will be fixed

Reply:
yes, that's what I was trying to mean :)